### DISCUSSION TITLE
nuxt 3 tailwindcss ui library problem

### DISCUSSION THREAD

**Discussion by Mamush-Meshesha** [nuxt/nuxt#25460] [Questions]
am using nuxt 3  with primevue tailwindcss ui , and i configure 
![Screenshot from 2024-01-27 15-31-33](https://github.com/nuxt/nuxt/assets/118711095/0bc35cbf-f003-44d9-b350-59d60adb8cc9)
on nuxt.config.ts file but while i use icon doesnt work only the background color is appear
![Screenshot from 2024-01-27 15-32-39](https://github.com/nuxt/nuxt/assets/118711095/c54ad14d-de0d-40d3-bb5a-52e95445f7bf)
could someone help me? please!!

**Comment by TheAlexLichter** [comment]
Best to ask that in https://github.com/nuxt/ui 👍
